xls/dslx/stdlib/std.x

```diff
@@ -1070,6 +1070,62 @@ fn test_vslice() {
     assert_eq(vslice<u32:0, u32:0>(u8:0xab), u1:1);
 }
 
+// Splits bits into (N most significant bits, the remaining least significant
+// bits).
+//
+// This function ensures that ensures that all bits of the argument are used.
+pub fn split_n_msbs<N: u32, X: u32, Z: u32 = {X - N}, FROM_START: s32 = {Z as s32}>
```

Comment:
One naming nit, since I think it's a little important to look at the consistency of stdlib function naming -- now that we have `keep_lsbs`, we should probably ditch the `_n_` for consistency, so it just becomes `split_msbs` and `split_lsbs`